admin_manual/installation/nginx.rst

```diff
@@ -123,13 +123,10 @@ webroot of your nginx installation. In this example it is
       location ^~ /.well-known {
           # The following 6 rules are borrowed from `.htaccess`
       
-          rewrite ^/\.well-known/host-meta\.json  /public.php?service=host-meta-json  last;
-          rewrite ^/\.well-known/host-meta        /public.php?service=host-meta       last;
-          rewrite ^/\.well-known/webfinger        /public.php?service=webfinger       last;
-          rewrite ^/\.well-known/nodeinfo         /public.php?service=nodeinfo        last;
-          
           location = /.well-known/carddav     { return 301 /remote.php/dav/; }
```

Comment:
Would be good to at some point then also migrate the *dav related well known endpoints

Reply:
Absolutely. @rullzer and I brainstormed about using the new API and having a simple handler in the dav app. Then installations that are hosted in the document root don't ever need to fiddle with the rewrites/redirects.